Tests/SwiftPrettyPrintTests/Combine/CombineExtensionInternalTests.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  CombineExtensionTests.swift
+//  SwiftPrettyPrintTests
+//
+//  Created by Yusuke Hosonuma on 2020/11/26.
+//
+
+import XCTest
+import Combine
+@testable import SwiftPrettyPrint
+
+// Note:
+// Original `print` operator output is:
+//
+// ```
+// receive subscription: ([[1, 2], [3, 4]])
+// request unlimited
+// receive value: ([1, 2])
```

Comment:
Default implementation is enclose in parentheses but remove it in our implementation because redundant.